user_guide_src/source/installation/upgrade_420.rst

```diff
@@ -33,6 +33,7 @@ Breaking Changes
 ****************
 
 - The ``system/bootstrap.php`` file no longer returns a ``CodeIgniter`` instance, and does not load the ``.env`` file (now handled in ``index.php`` and ``spark``). If you have code that expects these behaviors it will no longer work and must be modified. This has been changed to make `Preloading <https://www.php.net/manual/en/opcache.preloading.php>`_ easier to implement.
+- ``previous_url()`` has been changed to return only the URLs whose Content-Type was ``text/html``. Accordingly, the behavior of ``redirect()->back()`` has been changed.
```

Comment:
In the upgrade note, we write *what to do* when upgrading.
Since it seems unlikely that any developers were using non-HTML URLs, is there no need to list it?

Reply:
I see, I understood. I have removed it from the update notes 👍